src/CSVMsg.chpl

```diff
@@ -66,17 +66,14 @@ module CSVMsg {
 
         // open file and determine if header exists.
         var idx = 0;
-        var csvFile = open(filename, ioMode.r);
-        var reader = csvFile.reader();
+        var reader = openReader(filename);
         var lines = reader.lines().strip();
         if lines[0] == CSV_HEADER_OPEN {
             idx = 3; // set to first line after header
         }
 
         var col_delim: string = msgArgs.getValueOf("col_delim");
         var column_names = lines[idx].split(col_delim);
-        reader.close();
-        csvFile.close();
```

Comment:
we removed the reader close and file close here but in another section we only remove the file close